packages/utils/src/feature_toggle.rs

```diff
@@ -0,0 +1,414 @@
+        deps: &mut Extern<S, A, Q>,
+        _env: Env,
+        address: HumanAddr,
+    ) -> StdResult<HandleResponse> {
+        Self::_set_pauser(&mut deps.storage, address)?;
+
+        Ok(HandleResponse {
+            messages: vec![],
+            log: vec![],
+            data: Some(to_binary(&HandleAnswer::SetPauser {
+                status: ResponseStatus::Success,
+            })?),
+        })
+    }
+
+    pub fn handle_remove_pauser<S: Storage, A: Api, Q: Querier>(
+        deps: &mut Extern<S, A, Q>,
+        _env: Env,
+        address: HumanAddr,
+    ) -> StdResult<HandleResponse> {
+        Self::_remove_pauser(&mut deps.storage, address);
+
+        Ok(HandleResponse {
+            messages: vec![],
+            log: vec![],
+            data: Some(to_binary(&HandleAnswer::RemovePauser {
+                status: ResponseStatus::Success,
+            })?),
+        })
+    }
+
+    pub fn query_status<S: Storage, A: Api, Q: Querier>(
+        deps: &Extern<S, A, Q>,
+        features: Vec<String>,
+    ) -> QueryResult {
+        let mut status = vec![];
+        for f in features {
+            match Self::_get_feature_status(&deps.storage, f.clone())? {
+                None => {
+                    return Err(StdError::generic_err(format!(
+                        "invalid feature: {} does not exist",
+                        f
+                    )))
+                }
+                Some(s) => status.push(_FeatureStatus {
+                    feature: f,
+                    status: s,
+                }),
+            }
+        }
+
+        to_binary(&FeatureToggleQueryAnswer::Status { features: status })
+    }
+
+    pub fn query_is_pauser<S: Storage, A: Api, Q: Querier>(
+        deps: &Extern<S, A, Q>,
+        address: HumanAddr,
+    ) -> QueryResult {
+        let is_pauser = Self::_get_pauser(&deps.storage, address)?.is_some();
+
+        to_binary(&FeatureToggleQueryAnswer::IsPauser { is_pauser })
+    }
+
+    fn _stop<S: Storage>(storage: &mut S, features: Vec<String>) -> StdResult<()> {
+        for f in features {
+            Self::_set_feature_status(storage, f, FeatureStatus::Stopped)?;
+        }
+
+        Ok(())
+    }
+
+    fn _resume<S: Storage>(storage: &mut S, features: Vec<String>) -> StdResult<()> {
+        for f in features {
+            Self::_set_feature_status(storage, f, FeatureStatus::Resumed)?;
+        }
+
+        Ok(())
+    }
+
+    fn _get_pauser<S: ReadonlyStorage>(storage: &S, key: HumanAddr) -> StdResult<Option<u8>> {
+        let feature_store =
+            ReadonlyBucket::multilevel(&[PREFIX_FEATURE_TOGGLE, PREFIX_PAUSERS], storage);
+        feature_store.may_load(key.0.as_bytes())
+    }
+
+    fn _set_pauser<S: Storage>(storage: &mut S, key: HumanAddr) -> StdResult<()> {
+        let mut feature_store =
+            Bucket::multilevel(&[PREFIX_FEATURE_TOGGLE, PREFIX_PAUSERS], storage);
+        feature_store.save(key.0.as_bytes(), &1_u8 /* value is insignificant */)
```

Comment:
You could set a `bool` of `true` just to make it less arbitrary (and it takes just as much space)

Reply:
Hmmm yeah I guess it's less random

Reply:
Yeah you could do that. If it's not in the storage, translate it to `false`, if it is, unwrap the option?